### ISSUE TITLE
Wiki

### ISSUE THREAD

**Issue by bookofportals** [hubnomic/hubnomic#6]
I added a page to the wiki, and was wondering what people think. It's pretty much blank at the moment, but I think it can be used for historical records and stuff like that. Issues could be used for discussion.

**Comment by ojacobson** [comment]
I'd be partial to moving incidental game state (like the current scores) into the wiki, since the process for modifying the game state is largely a pure consequence of the execution of rules and is unlikely to have any interesting discussion attached to it, for the time being. I didn't put a lot of thought into the question of where to put scores: I put them in the repo because Things Go In The Repo, rather than out of any principled stance on data storage.

(We have an unsettled score, for @bwarren2's proposal #7, so this is of more than passing interest to me.)

**Comment by bookofportals** [comment]
I'd prefer to keep them in the repo. I like the idea of keeping non-rule stuff in there, and it can be inserted without a pull request. I'd say we should use the wiki for more... informal, textual records. Essays and the like.

**Comment by bwarren2** [comment]
So score maintenance is Owen hacking on master for now?

**Comment by bookofportals** [comment]
@bwarren2: Yep. Is that a problem? If there's something that could be controversial, we can always use a unregulated pull request, which by custom (the one time that it's happened) passes with a simple majority vote.

**Comment by bwarren2** [comment]
Nope, I am fine with it as long as Owen does not mind.

**Comment by bookofportals** [comment]
Anyone can always do it. Even if we make an office of scorekeeper, it'll probably be a ceremonial, confirm-what-others-have-done kind of thing.

**Comment by ojacobson** [comment]
I weakly prefer allowing anyone to do so, as long as keeping score is unregulated, over there being a designated score-keeping job: there is a hypothetically-correct source of truth (the repository), so there's no need for a designated gatekeeper for the data in it. I assume we can all be trusted to do arithmetic…

That said, for practical purposes I _am_ keeping score - I just scored two more proposals.
